GANDLF/config_manager.py

```diff
@@ -497,7 +499,7 @@ def _parseConfig(
                                 "max"
                             ] = sys.float_info.max
                 elif key in thresholdOrClipDict:
-                    sys.exit("Use only 'threshold' or 'clip', not both")
+                    logging.error("Use only 'threshold' or 'clip', not both")
```

Comment:
Thanks for the update! Altough, should we completly disregard throwing an error here? My intention was to rather just log the error via logger in addition to throwing an error, be it assertion or sys.exit. Should we resign from this functionality? If so, then it is fine

Reply:
Hm. I was going to propose just raising an exception:
```suggestion
                    raise ValueError("Use only 'threshold' or 'clip', not both")
```
However I just noticed we don't log exceptions to the debug file (@benmalef FYI). I believe, we should fix that and then raising exception would be the cleanest solution - both logged & exiting the gandlf command with exit_code 1

Reply:
Brilliant, let's make it for the next PR before tagging the new API changes.